src/App.js

```diff
@@ -1,12 +1,81 @@
+import {useEffect, useState} from 'react';
 import React from 'react';
 import './App.css';
+import axios from "axios";
+import Home from "./Component/Home/Home";
+import Button from "./Component/Button/Button";
+import logo from "./assets/Pokemon.png"
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemons, setPokemons] = useState([]);
```

Comment:
`pokemons` zal uiteindelijk een object zijn. Hierbij past een default waarde van `undefined` beter waardoor je ook een duidelijk verschil krijgt tussen of iets wel of niet is ingeladen
```suggestion
    const [pokemons, setPokemons] = useState();
```